sql/src/test/java/org/apache/druid/sql/calcite/CalciteWindowQueryTest.java

```diff
@@ -219,6 +219,28 @@ public void windowQueryTest() throws Exception
     }
   }
 
+  @Test
+  @SuppressWarnings("unchecked")
+  public void windowQueryTestWithCustomContextMaxSubqueryBytes() throws Exception
```

Comment:
I wonder if instead of running all testcases in this mode as well - wouldn't it be better to introduce the ability to specify querycontext keys in the `sqlTest` files; so that `rank_handling.sqlTest` can set it

Reply:
Good idea, I have other tests in flight as well, we can merge this in and I'll send out a refactoring for the test cases